doc/qbk/06_websocket/02_handshaking.qbk

```diff
@@ -150,6 +150,19 @@ stream:
 
 [code_websocket_2_5]
 
+[heading Sub Protocols]
+
+The Websocket protocol understands the concept of sub-protocols. If the client
+is requesting one of a set of sub-protocols it will set the header `Sec-Websocket-Protocol`
```

Comment:
It is spelled **Sec-WebSocket-Protocol** note capitalization. And it does not use a monospaced font, since it is not a C++ identifier.